doc-content/jbpm-docs/src/main/asciidoc/ReleaseNotes/Release.7.17.0.Final-section.adoc

```diff
@@ -26,3 +26,48 @@ Furthermore, it is also possible for users to interactively select nodes in the
 on the left for a more specific visualization of the node data.
 
 image:ReleaseNotes/process_admin_operations_7.17.0.png[align="center", title="Process instance diagram"]
+
+== Service task repository integrated with Business Central
+
+Service task repository (aka work item repository) has been integrated into Business Central. It provides two areas
+to manage service tasks
+
+* Global administration - accessible via settings page with dedicated `Service Task Administration` tile
+* Project level management - accessible via project settings
+
+Global administration allows process administrators to define what service tasks can be used within projects
+by either enabling or disabling them. Most of service tasks available in repository are disabled by
+default except those that come with jBPM out of the box e.g. REST, Email, Decision, Business rule.
```

Comment:
allows - enables
Most service tasks...in the repository...
e.g.- for example
Decision, and Business rule - should rule be Rules?